crates/vm/levm/src/errors.rs

```diff
@@ -190,6 +190,8 @@ pub enum InternalError {
     InvalidPrecompileAddress,
     #[error("Spec Id doesn't match to any fork")]
     InvalidSpecId,
+    #[error("EIP7702 error")]
+    EIP7702Internal(#[from] EIP7702Error),
```

Comment:
This indirection would only make sense if `EIP7702Error` had more errors. Based on the error raise reason: 
```suggestion
    #[error("Account should had been delegated")]
    AccountNotDelegated,
```

Reply:
makes sense